src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparatorRecordsAndCompactCtors.java

```diff
@@ -0,0 +1,67 @@
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.whitespace.emptylineseparator; // violation
```

Comment:
This is not related to the issue. Will this violation gone if you add an empty line between the package declaration and the header?

Reply:
Yes, the violation will be gone if these lines are separated? Should they be? This is the format I've followed with all other inputs, so I left it even though it was unrelated, for consistency.

Reply:
I'm OK to leave it as is.